lib/src/configuration.dart

```diff
@@ -241,6 +242,14 @@ extension FlexibleSyncConfigurationInternal on FlexibleSyncConfiguration {
   set sessionStopPolicy(SessionStopPolicy value) => _sessionStopPolicy = value;
 }
 
+class SyncFollowerConfiguration extends Configuration {
```

Comment:
We can come up with a more meaningful name for SyncFollowerConfiguration. But I have no ideas.

Reply:
Yes me too.. I was hoping you could help me out 😏

Reply:
Any of these you like:
- `SecondarySyncConfiguration`,
- `SecondaryProcessConfiguration`,
- `SyncSlaveConfiguration`,
- `SyncSecondaryConfiguration`,
- `SyncReplicaConfiguration`,
- `ReplicaConfiguration`,
- `SyncStandbyConfiguration`,
-  `StandbyConfiguration`

Reply:
`SessionlessSyncConfiguration`? 😄